.pubignore

```diff
@@ -0,0 +1,55 @@
+# See https://dart.dev/guides/libraries/private-files
```

Comment:
If `.gitignore` and `.pubignore` are identical, we can omit `.pubignore`.

If they're not identical, can we generate the one from the other with a script in `tool/`? Or maybe generate both? That way we make sure it doesn't go out of sync and we accidentally start publishing too many files to pub. (E.g. of something gets added to `.gitignore` but not to `.pubignore`.)